app/views/home/index.html.erb

```diff
@@ -39,9 +39,12 @@
       </div>
     </form>
     <p class="usa-prose-body">
-      <%= link_to 'Browse all practices', search_path, class: 'usa-link' %>
+      <%= link_to 'Browse all innovations', search_path, class: 'dm-primary-link' %>
```

Comment:
I set the link back.. not sure how that happened... so the the change to innovation branch is branched off Master.. so I"m still going to have to go through the feature branch and update up any new entries for 'practice'..... so I'm just moving forward with 'innovation' since this branch is going back in to feature.